image-index.md

```diff
@@ -55,7 +55,10 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`os.version`** *string*
 
-        This OPTIONAL property specifies the operating system version, for example `10.0.10586`.
+        This OPTIONAL property specifies the version of the operating system targeted by the referenced blob.
+        Implementations MAY refuse to use manifests where `os.version` is not known to work with the host OS version.
+        The values are implementation-defined when `os` is `windows`. e.g. `10.0.14393.1066`.
+        The values for other `os`-es SHOULD be submitted to this specification for standardization.
```

Comment:
20dcb7a9 → de499b4e dropped implementation-defined for non-Windows `os`.  I think the full section should read:

```
This OPTIONAL property specifies the version of the operating system targeted by the referenced blob.
Implementations MAY refuse to use manifests where `os.version` is not known to work with the host OS version.
Valid values are implementation-defined.
```

Once we make the values implementation-defined, there's no clawing it back into the spec without risking breaking changes, so I don't think we should bother asking for submission for standardization.